app/workers/rapid_ready_for_decision/rrd_processor.rb

```diff
@@ -11,7 +11,7 @@ def initialize(form526_submission)
 
     def run
       assessed_data = assess_data
-      return if assessed_data.nil?
+      return @form526_submission.save_metadata(offramp_reason: 'insufficient_data') if assessed_data.nil?
```

Comment:
Explicitly capture `insufficient_data` as the reason, rather than assume that reason as the default.

Reply:
Love this change, the meaning of "insufficient_data" confused me at first.